CMakeLists.txt

```diff
@@ -231,7 +231,7 @@ if(ENABLE_TESTING)
 endif()
 
 # Make scripts needed for testing available in an out-of-source build.
-if (NOT ${CMAKE_CURRENT_BINARY_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
+if (ENABLE_TESTING AND NOT ${CMAKE_CURRENT_BINARY_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
There is an `if(ENABLE_TESTING)` block right before this section, so now we have:

```
if(enable_testing)
    <1st block>
endif()

if(enable_testing AND <other condition>)
    <2nd block>
endif()
```

Why not moving things around and have:

```
if(enable_testing)
    <1st block>
    if(NOT <other condition>)
        <2nd block>
    endif()
endif()
```

This way we have all the testing related rules in a single block, which I think we should try to maintain for the sake of code's readability.

Reply:
Yeah. This is better. Thanks @k-stachowiak.